src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -297,37 +298,21 @@ private String getDemoCourseIdRoot(String instructorEmail){
     private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId, int maximumIdLength){
         final boolean isFirstCourseId = instructorEmailOrProposedCourseId.contains("@");
         if(isFirstCourseId){
-            return trimCourseIdToMaximumLengthIfNecessary(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
-                    , maximumIdLength);
+            return StringHelper.truncateHead(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
+                                            , maximumIdLength);
```

Comment:
@damithc I think line break before `,` is fine. We are using line break before Operators everywhere. If we will use line break after  `,`  in this case, it will be inconsistent.